.env.example

```
@@ -1 +1,3 @@
-VITE_UPSTAGE_API_KEY=up_*********KwzE
\ No newline at end of file
+VITE_UPSTAGE_API_KEY=up_*********KwzE
+VITE_UPSTAGE_API_URL=https://api.upstage.ai/v1/solar/chat/completions
+VITE_UPSTAGE_SYSTEM_PROMPT="You are a helpful assistant."
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)